File: NQS/NQS/NQS.webapp/ClientApp/src/app/models/Organizacao.ts
import { PessoaDados } from './Pessoa';

export class Organizacao {
    id?: string;
    nome: string;
    foto: string;
    descricao: string;
    localizacao: string;
    idUsuario: string;
    idPessoas: Array<string>;
    dataCriacao: Date;
    responsavelCriacao: string;
    dataEdicao?: Date;
    responsavelEdicao?: string;
    pessoas: PessoaDados[];
}

export class OrganizacaoDadosBasicos {
    idOrganizacao: string;
    nomeOrganizacao: string;
}

export class UsuarioOrganizacao {
    idOrganizacao: string;
    idUsuario: string;
    dataCriacao: Date;
    responsavelCriacao: string;
}

File: NQS/NQS/NQS.Entity/Entity/Organizacao.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Leega.Entity.Entity
{
    public class Organizacao : BaseEntity
    {
        [Column(TypeName = "VARCHAR(100)")]
        public string Nome { get; set; }
        public string Foto { get; set; }
        [Column(TypeName = "VARCHAR(1000)")]
        public string Descricao { get; set; }
        [Column(TypeName = "VARCHAR(100)")]
        public string Localizacao { get; set; }
        public DateTime DataCriacao { get; set; }
        public Guid ResponsavelCriacao { get; set; }
        public DateTime? DataEdicao { get; set; }
        public Guid? ResponsavelEdicao { get; set; }
        public virtual ICollection<OrganizacaoUsuario> Pessoas { get; set; }
        public virtual ICollection<Pessoa> UltimosAcessos { get; set; }
    }
}